lib/file_store/s3_store.rb

```diff
@@ -8,11 +8,16 @@ module FileStore
 
   class S3Store < BaseStore
     TOMBSTONE_PREFIX ||= "tombstone/"
+    MULTISITE_TOMBSTONE_PREFIX ||= File.join("uploads", "tombstone", RailsMultisite::ConnectionManagement.current_db, "/")
```

Comment:
This doesn't look right.... When the constant is loaded, it'll always be `default`